cms/updates/0.0.4-remove-sources.js

```diff
@@ -0,0 +1,63 @@
+const keystone = require("keystone"),
+	async = require("async"),
+	log4js = require("log4js");
+	newSources = require("./sources2.json");
+
+const Source = keystone.list("Source");
+
+const logger = log4js.getLogger();
+
+const removeSource = async (source, done) => {
+	logger.info(`Source is ${source.title} with id ${source._id}`);
+
+	let existsInNewSourceList = false;
+
+	newSources.forEach(element => {if(element.oldEPiServerId == source.oldEPiServerId){existsInNewSourceList = true}})
+
+	if(existsInNewSourceList == true){
+		logger.info(`Not deleting ${source.title} as it exists in the new source list`)
+	}else{
+		logger.info(
```

Comment:
If you `return;` in the if statement above then all this doesn't need to be inside an else which makes it easier to read as you've got one less level of nesting